conf/hubble

```diff
@@ -88,7 +88,7 @@ fileserver_backend:
 #      flagfile: /path/to/confbase/osquery/osquery.flags
 #      logdir: /var/log/hubble_osquery
 #      databasepath: /var/cache/hubble/osquery
-#      pidfile: /var/run/hubble_osquery/osquery.pidfile
+#      pidfile: /var/run/osquery.pidfile
```

Comment:
I don't think this is a good idea. It is possible that someone has installed a different copy of osquery on their machine and then they install Hubble. The original installation of osquery would write its pidfile to the location you mentioned. If in this case, hubble's osqueryd also writes its pidfile to the same location then that would cause some trouble.

Reply:
I agree. Though, under systemd, it probably wouldn't cause all that much trouble. Systemd tracks the processes in a cgroup, so it would hardly care about the contents of a pidfile. It's more like just confusing for the humans looking in the file.

Reply:
@zachsis could you explain the thinking here? Is this a problem with the /var/run/hubble_osquery dir being absent ... or maybe some other thing I'm not thinking about?

Reply:
if we account for creating the `/var/run/hubble_osquery` at some point during the install, I think that would be sufficient. The idea is that the boiler plate configs should point to a folder that exists so there is less manual intervention by the user to set that up. i.e. if you uncomment line 91, it should work without additional troubleshooting.

Reply:
how about we change it to a lesss generic pidfile name and still use `/var/run/` as the path unless you guys would rather account for creating `/var/run/hubble_osquery` on install? https://github.com/hubblestack/hubble/pull/896/commits/02c4c598161e1264258744c22bd8e6657808d7c3

Reply:
@zachsis @MoodyMudit where did we leave off on this? I don't think we need a dir in /var/run; but if we do use a dir like that, we should at least make sure it exists in each of the systemd/init-d startups (or whatever). I was about to merge this (because it's just a comment in a default config file); but it's blocked waiting to see the outcome of this change request.

Reply:
If we choose to have `/var/run/osquery.pidfile` folder as default, then this can overlap with any other osquery installation on the system (although having two osquery would be highly unlikely). But in case someone has two osqueries installed and writing pid to the same file then the one corresponding to Hubble would face problems. This is because, Hubble monitors that the pid in the pidfile is same as the pid of the process. In case it is not the same, Hubble would restart osquery. 
This is the reason we chose to have a separate folder `hubble_osquery` too keep the pidfile separate. 
I agree we should create the folder `/var/run/hubble_osquery/` to remove that manual folder creation step. 
@zachsis Let me know if this addressed properly. 
As such, this PR can be closed, and a new one (or even modify this one) with the folder creation should be raised.